pkg/server/plugin/notifier/v1.go

```diff
@@ -0,0 +1,83 @@
+package notifier
+
+import (
+	"context"
+
+	"github.com/spiffe/go-spiffe/v2/spiffeid"
+	notifierv1 "github.com/spiffe/spire-plugin-sdk/proto/spire/plugin/server/notifier/v1"
+	"github.com/spiffe/spire-plugin-sdk/proto/spire/plugin/types"
+	"github.com/spiffe/spire/pkg/common/plugin"
+	"github.com/spiffe/spire/proto/spire/common"
+	"google.golang.org/grpc/codes"
+)
+
+type V1 struct {
+	plugin.Facade
+	notifierv1.NotifierPluginClient
+}
+
+func (v1 *V1) NotifyAndAdviseBundleLoaded(ctx context.Context, bundle *common.Bundle) error {
+	pluginBundle, err := bundleFromCommonProto(bundle)
+	if err != nil {
+		return v1.Errorf(codes.InvalidArgument, "bundle is malformed: %v", err)
```

Comment:
This path is not exercised by the tests, we may add a case relatively easily I think.

Reply:
SGTM!